manifests/m/MythicsoftLtd/FileLocator/9.1.3389.1/MythicsoftLtd.FileLocator.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MythicsoftLtd.FileLocator
+PackageVersion: 9.1.3389.1
+Installers:
+- InstallerUrl: https://download.mythicsoft.com/flp/3389/filelocator_3389.exe
+  Architecture: x86
+  InstallerType: nullsoft
+  InstallerSha256: 98B5654ABE82CEB0F56E4C8AF9F7AFE928D44F113DF2DCA83231BB1575817242

```

Comment:
```suggestion
- InstallerUrl: https://download.mythicsoft.com/flp/3389/filelocator_x64_msi_3389.zip
  Architecture: x64
  InstallerType: wix
  InstallerSha256: 1A187A8F618E254FEA67BBFD52023CD43803529FB127702E71E010603527F99C
```

Reply:
The `x86` MSI is a 16-bit application that won't install on 64-bit OSes. 
The `x64` MSI isn't a valid package.
![image](https://user-images.githubusercontent.com/114952565/236030650-2282ac45-13f1-48bc-955a-1a0245377f5f.png)